### PR TITLE
Update map visualizations

### PR DESCRIPTION
**Pull request by necabotheking**
## What does this pull request add?

Updates the map visualizations and formats the Dash app.
Fixes the 'make visualize' command.
Unused parts of app.py are commented out and will be taken out later.

## Are there any technical things that should be noted for this PR?
N/A

## How was it tested?
- [ ] Wrote test cases.
- [x] Previously working test cases are still passing.

```
reproductive_rights_data_project/test/test_api.py::test_get_api_data PASSED
reproductive_rights_data_project/test/test_api.py::test_get_state_zip_code_geo_json_first_call PASSED
reproductive_rights_data_project/test/test_api.py::test_get_state_zip_code_geo_json_not_first_call PASSED
reproductive_rights_data_project/test/test_api.py::test_set_default_types PASSED
reproductive_rights_data_project/test/test_api.py::test_fill_in_missing_data PASSED
reproductive_rights_data_project/test/test_api.py::test_add_missing_states PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_make_row_dicts PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_set_default_types PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_clean_ansirh PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_translate_code_to_state PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_split_by_state PASSED
reproductive_rights_data_project/test/test_data_handling.py::test_split_by_zip PASSED
reproductive_rights_data_project/test/test_util.py::test_to_json PASSED
```
- [x] I ran `pylint` and have made the bulk of changes that it has requested.
- [x] I ran the `black` formatter and pushed the changes.

### CODE REVIEW

**Review by michplunkett — COMMENTED** [review]
It looks quite good! I removed a lot of comments, did some formatting, and made a public function that was only being called internally private.

Before we merge this, I'd like to see a removal of some of the copy and pasted code between the three charts.

File: reproductive_rights_data_project/visualization/service.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 """
 from reproductive_rights_data_project.visualization.app import (
     DASH_INSTANCE,
     build_dash,
 )
 from reproductive_rights_data_project.visualization.maps.usa_country import (
-    USAMap as USAMap,
-)
-from reproductive_rights_data_project.visualization.maps.usa_state import (
-    USAState as USAState,
```
Comment (michplunkett) [inline reproductive_rights_data_project/visualization/service.py:14]: The `as` here is redundant since it's the same  name.

**Review by chanteriam — APPROVED** [review]
Everything looks good to me (:

**Review by michplunkett — APPROVED** [review]
Looks gewwwwd

**Pull request merged**